README.md

```diff
@@ -13,6 +13,13 @@ Features:
    * manage registry credentials for private and public repos
    * tag the image built - use any Jenkins env. variables.
 
+## Upgrading
+
+In versions 1.0+ the plugin uses [https://github.com/jenkinsci/docker-commons-plugin](docker-commons-plugin)
```

Comment:
Can link to [wiki](https://wiki.jenkins-ci.org/display/JENKINS/Docker+Commons+Plugin).